core/src/avm2/qname.rs

```diff
@@ -77,34 +77,16 @@ impl<'gc> QName<'gc> {
     /// NAMESPACE.LOCAL_NAME (Where the LAST dot is used to split the namespace & local_name)
     /// LOCAL_NAME (Use the public namespace)
     pub fn from_qualified_name(name: AvmString<'gc>, activation: &mut Activation<'_, 'gc>) -> Self {
```

Comment:
can you please add a comment to the method, saying that it explicitly doesn't support `.<T>`?